src/query/ast/src/ast/expr.rs

```diff
@@ -835,6 +835,7 @@ pub struct FunctionCall {
     pub name: Identifier,
     pub args: Vec<Expr>,
     pub params: Vec<Expr>,
+    pub window_ignore_null: Option<bool>,
```

Comment:
How about put `window_ignore_null` in the `Window` struct as a field `ignore_null`? This can make the definition of `FunctionCall` look simpler.

Reply:
Now the window only contain `over` part. But ignore nulls is function part options.

Reply:
```
#[derive(Debug, Clone, PartialEq, EnumAsInner, Drive, DriveMut)]
pub enum Window {
    WindowReference(WindowRef),
    WindowSpec(WindowSpec),
}
```

I think ignore nulls is not suitable put in enum Window.

Reply:
we can define `Window` as a struct, and use current enum as a field in `Window`.

Reply:
Done. Please review.